hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -0,0 +1,89 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "rv_dm",
+  clock_primary: "clk_i",
+  bus_interfaces: [
+    { protocol: "tlul", direction: "host",   name: "sba" }
+    { protocol: "tlul", direction: "device", name: "regs" }
+    { protocol: "tlul", direction: "device", name: "rom" }
```

Comment:
@tjaychen maybe we can reuse this mechanism for the Ibex CSRs (i.e., make `rv_core_ibex` a comportable module)?

Reply:
Yes, totally!